CMakeLists.txt

```diff
@@ -25,6 +25,11 @@ if(POLICY CMP0064)
 endif()
 
 set(PROJECT_NAME "Cockatrice")
+set(PROJECT_VERSION_MAJOR "2")
+set(PROJECT_VERSION_MINOR "3")
+set(PROJECT_VERSION_PATCH "15")
+# Set to true to append git commit hash to version number
+set(PROJECT_IS_PRERELEASE True)
```

Comment:
The idea was to:
 * update the version number and set PROJECT_IS_PRERELEASE to false just before the release,
 * then tag the release in order to get it built by travis/appveyor
 * if everything is in order, publish it as a release
 * after the release, update the version number to "next dev version" and set PROJECT_IS_PRERELEASE back to True